src/main.tf

```diff
@@ -16,22 +16,12 @@ resource "azurerm_storage_account" "main" {
   account_tier                      = "Standard"
   account_kind                      = "StorageV2"
   access_tier                       = "Hot"
-  account_replication_type          = var.redundancy.replication_type
+  account_replication_type          = var.redundancy.data_protection ? var.redundancy.replication_type : "LRS"
```

Comment:
For this, we should rip `LRS` out of here then too: https://github.com/massdriver-cloud/azure-storage-account-application-assets/blob/main/massdriver.yaml#L148

Reply:
This is an unrelated change and I don't currently have context on what you're talking about. I brought the current default in here because I was getting tired of messing with my `_params.auto.tfvars.json` all the time when provisioning locally. 

If removing `LRS` from this PR gets a thumbs up from ya then okie dokie, otherwise, can you please provide more context? I assume this an an availability thing? I picked it because it sounded like the simplest, that it might be available everywhere.

Reply:
This change means that they can't even pick GRS as a replication type, even if they keep data protection off. GRS and RAGRS are values exposed to the user, and this change overrides the users selection of those 2 values

Reply:
The massdriver.yaml that I linked shows how those params are set up

Reply:
We'll come back to it, open a PR